src/components/Layout/FeatureItem/FeatureTwo.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
```

Comment:
The React import was previously required to be specified in all files. But from React 17, you dont need to specify explicitly. It does on its own.